assets/app/scripts/controllers/project.js

```diff
@@ -8,21 +8,47 @@
  * Controller of the openshiftConsole
  */
 angular.module('openshiftConsole')
-  .controller('ProjectController', function ($scope, $routeParams, DataService, AuthService, $filter, LabelFilter) {
+  .controller('ProjectController', function ($scope, $routeParams, DataService, AuthService, $filter, LabelFilter, $location) {
 
     $scope.projectName = $routeParams.project;
     $scope.project = {};
     $scope.projectPromise = $.Deferred();
     $scope.projects = {};
+    $scope.alerts = {};
     $scope.renderOptions = {
       hideFilterWidget: false
     };
 
     AuthService.withUser().then(function() {
-      DataService.get("projects", $scope.projectName, $scope).then(function(project) {
-        $scope.project = project;
-        $scope.projectPromise.resolve(project);
-      });
+      DataService.get("projects", $scope.projectName, $scope, {noNotification: true}).then(
+        // success
+        function(project) {
+          $scope.project = project;
+          $scope.projectPromise.resolve(project);
+        },
+        // failure
+        function(e) {
+          $scope.projectPromise.reject();
```

Comment:
doesn't seem to get here, threw an alert that was never fired, redirecting to login as expected.